### ISSUE TITLE
Missing content 

### ISSUE THREAD

**Issue by aak2054** [kevinaangstadt/CS340-S23-Notes#79]
is there a way if we Identify any important content that is missing from the lecture notes, like if a topic was covered in class but is not mentioned in the notes. we can mark down that to be added

**Comment by CharlesReinhardt** [comment]
Yeah good idea, it would definitely be good to mark that down somehow when we see it. We could potentially raise an issue like this one that mentions the missing content, and someone could submit a PR that modifies that day's lecture notes to include the important but missing content.

**Comment by jaxlub** [comment]
I think also part of this project is a contributing to other peoples MD files and making a PR. This would provide a good opportunity to do so either in the form of adding the missing note or just commenting it in for the owner to see and correct later.
